checked/bytes.go

```diff
@@ -54,6 +54,7 @@ func NewBytes(value []byte, opts BytesOptions) Bytes {
 		value: value,
 	}
 	b.SetFinalizer(b)
+	b.TrackObject(b.value)
```

Comment:
Can we add a global like `trackLeak` in `debug.go` so we can avoid doing this all the time and enable only when we want to? i.e. similar to in `ref.go`:

```go
// ref.go
if traceback {
  tracebackEvent(...)
}
```

```go
// debug.go
var (
  traceLeak = false
)

func SetTraceLeak(value bool) {
  traceLeak = value
}
```